hw/top_earlgrey/BUILD

```diff
@@ -338,6 +352,8 @@ silicon(
     libs = [
         "//sw/device/lib/arch:boot_stage_owner",
         "//sw/device/lib/arch:silicon",
+        "//hw/top_earlgrey/sw/autogen:devicetables",
+        "//hw/top_earlgrey/sw/cw310:devicetables",
```

Comment:
This cw310 table is everywhere just for playing around. The different board-level environments should have their own pin-out and clock definitions.